### PR TITLE
Update enterprise admin section

### PR DESCRIPTION
**Pull request by kosyfrances**
When tfe v202203-1 will be released, the [active/active](https://www.terraform.io/enterprise/admin/infrastructure/active-active) commands will also be supported on standalone installation.
This PR updates the doc sections referencing that to be in line with the changes, alongside renaming the URL for that page.
Redirects are also updated accordingly.

https://www.terraform.io/enterprise/admin/infrastructure/active-active --> https://terraform-website-lya0aige2-hashicorp.vercel.app/enterprise/admin/infrastructure/admin-cli

### CODE REVIEW

File: content/enterprise/admin/infrastructure/admin.mdx [hunk 1]
```diff
@@ -1,16 +1,12 @@
 ---
-page_title: Active/Active - Infrastructure Administration - Terraform Enterprise
+page_title: Infrastructure Administration - Terraform Enterprise
 ---
 
-# Terraform Enterprise Active/Active
+# Terraform Enterprise Admin
 
-When your organization requires increased reliability or performance from Terraform Enterprise that your current single application instance cannot provide, it is time to scale to the Active/Active architecture. Please refer to  [Terraform Enterprise Active/Active](/enterprise/install/automated/active-active) for more information on moving to Active/Active.
+The [Active/Active](/enterprise/install/automated/active-active) operational mode disables use of the Replicated Admin Console.  To accommodate this, there are now admin commands to facilitate configuration changes, safe application stops, and producing support bundles. There is a new container in the TFE architecture to support this called "tfe-admin". You must login to a node in the Active/Active cluster using SSH to run these commands from the command line.
```
Comment (laurapacilio) [inline content/enterprise/admin/infrastructure/admin.mdx:7]: Do we need to tell folks here about the new container, or can we just tell them how to log in and run the commands? This shows my lack of knowledge of TFE - please let me know if that should be added back in. BUT I made a few edits to the rest to make it flow better!

```suggestion
The [Active/Active](/enterprise/install/automated/active-active) operational mode disables the Replicated Admin Console. Instead, it provides admin commands to change the configuration, stop the application safely, and produce support bundles. You must use SSH to log in to a node in the Active/Active cluster to run these commands from the command line. 
```

File: content/enterprise/install/automated/active-active.mdx [hunk 2]
```diff
@@ -6,12 +6,14 @@ page_title: Automated Installation - Active/Active - Terraform Enterprise
...
 
+~> **Note:** The discontinued clustered version of Terraform Enterprise is no longer supported, and we strongly advise all customers to move to an Active/Active installation. Please contact your support representative if you need assistance.
+
 However, the benefits that come with the Active/Active architecture need to be weighed against increasing operation complexity:
 
 * Hard requirement of a completely automated installation
 * Observability concerns when monitoring multiple instances
 * Custom automation required to manage the lifecycle of application nodes
-* [New CLI-based commands](/enterprise/admin/infrastructure/active-active) vs Replicated Admin Console for administration
+* [New CLI-based commands](/enterprise/admin/infrastructure/admin) vs Replicated Admin Console for administration
```
Comment (laurapacilio) [inline content/enterprise/install/automated/active-active.mdx:16]: They won't be "new" for very long! :-) 

```suggestion
* [CLI-based commands](/enterprise/admin/infrastructure/admin) for administration instead of the Replicated Admin Console
```

File: data/enterprise-nav-data.json [hunk 3]
```diff
@@ -204,8 +204,8 @@
             "path": "admin/infrastructure/backup-restore"
           },
           {
-            "title": "Active/Active",
```
Comment (laurapacilio) [inline data/enterprise-nav-data.json:207]: I'm thinking through this, and I'm wondering if just "Admin" will be confusing for users, since this is underneath a broader section called administration. What if we called it Admin CLI Commands or something similar? That's really what this is about, right? I just want to make sure we're giving users as much information as possible from the name so they know what they're going to get when they click.

Comment (kosyfrances) [reply data/enterprise-nav-data.json:207]: Admin CLI Commands sounds better. Thanks!

File: content/enterprise/admin/infrastructure/admin.mdx [hunk 4]
```diff
@@ -1,16 +1,12 @@
 ---
-page_title: Active/Active - Infrastructure Administration - Terraform Enterprise
+page_title: Infrastructure Administration - Terraform Enterprise
 ---
 
-# Terraform Enterprise Active/Active
+# Terraform Enterprise Admin CLI commands
```
Comment (laurapacilio) [inline content/enterprise/admin/infrastructure/admin.mdx:5]: We always want to use title case when we have titles of any level :-) 

```suggestion
# Terraform Enterprise Admin CLI Commands
```

File: content/enterprise/admin/infrastructure/admin.mdx [hunk 5]
```diff
@@ -1,16 +1,12 @@
...
 
-When your organization requires increased reliability or performance from Terraform Enterprise that your current single application instance cannot provide, it is time to scale to the Active/Active architecture. Please refer to  [Terraform Enterprise Active/Active](/enterprise/install/automated/active-active) for more information on moving to Active/Active.
+The [Active/Active](/enterprise/install/automated/active-active) operational mode disables the Replicated Admin Console. Instead, it provides admin commands to change the configuration, stop the application safely, and produce support bundles. You must use SSH to log in to a node in the Active/Active cluster to run these commands from the command line.
```
Comment (laurapacilio) [inline content/enterprise/admin/infrastructure/admin.mdx:7]: Just thinking through flow - that last sentence felt a bit wordy. Okay also - should we say "Active/Active and Standalone?  It's both for the latest versions onward, right?  

```suggestion
The [Active/Active](/enterprise/install/automated/active-active) operational mode disables the Replicated Admin Console. Instead, it provides admin CLI commands to change the configuration, stop the application safely, and produce support bundles. You must use SSH to log in to a node in the Active/Active cluster to run these commands.
```

File: content/enterprise/install/automated/active-active.mdx [hunk 6]
```diff
@@ -6,12 +6,14 @@ page_title: Automated Installation - Active/Active - Terraform Enterprise
 
 When your organization requires increased reliability or performance from Terraform Enterprise that your current single application instance cannot provide, it is time to scale to the Active/Active architecture.
 
+~> **Note:** The discontinued clustered version of Terraform Enterprise is no longer supported, and we strongly advise all customers to move to an Active/Active installation. Please contact your support representative if you need assistance.
+
 However, the benefits that come with the Active/Active architecture need to be weighed against increasing operation complexity:
```
Comment (laurapacilio) [inline content/enterprise/install/automated/active-active.mdx:11]:
```suggestion
Before scaling to Active/Active, you should weigh its benefits against increasing operational complexity. Specifically, consider the following aspects of the Active/Active architecture:
```

File: content/enterprise/install/automated/active-active.mdx [hunk 7]
```diff
@@ -6,12 +6,14 @@ page_title: Automated Installation - Active/Active - Terraform Enterprise
 
 When your organization requires increased reliability or performance from Terraform Enterprise that your current single application instance cannot provide, it is time to scale to the Active/Active architecture.
 
+~> **Note:** The discontinued clustered version of Terraform Enterprise is no longer supported, and we strongly advise all customers to move to an Active/Active installation. Please contact your support representative if you need assistance.
```
Comment (laurapacilio) [inline content/enterprise/install/automated/active-active.mdx:9]:
```suggestion
!> **Warning:** The discontinued clustered version of Terraform Enterprise is no longer supported, and we strongly advise all customers to move to an Active/Active installation. Please contact your support representative if you need assistance.
```

File: content/enterprise/releases/2022/v202201-1.mdx [hunk 8]
```diff
@@ -25,7 +25,7 @@ APPLICATION LEVEL FEATURES:
 1. Container resource usage metrics are now available in Prometheus format.
 1. The private registry UI now displays a warning message for old versions of provider documentation with a link to the latest version. It also includes an **On this Page** outline for provider documentation that lets you navigate more quickly between sections. 
 1. Added an outline to the public provider documentation page
-1. The [new `tfe-admin rotate-encryption-password` command](https://www.terraform.io/enterprise/admin/infrastructure/active-active) rotates the encryption password on active/active installations of Terraform Enterprise.
+1. The [new `tfe-admin rotate-encryption-password` command](https://www.terraform.io/enterprise/admin/infrastructure/admin) rotates the encryption password on active/active installations of Terraform Enterprise.
```
Comment (laurapacilio) [inline content/enterprise/releases/2022/v202201-1.mdx:28]: Should Active/Active be capitalized here?

File: content/enterprise/releases/2022/v202201-2.mdx [hunk 9]
```diff
@@ -32,7 +32,7 @@ APPLICATION LEVEL FEATURES:
 1. Container resource usage metrics are now available in Prometheus format.
 1. The private registry UI now displays a warning message for old versions of provider documentation with a link to the latest version. It also includes an **On this Page** outline for provider documentation that lets you navigate more quickly between sections. 
 1. Added an outline to the public provider documentation page
-1. The [new `tfe-admin rotate-encryption-password` command](https://www.terraform.io/enterprise/admin/infrastructure/active-active) rotates the encryption password on active/active installations of Terraform Enterprise.
+1. The [new `tfe-admin rotate-encryption-password` command](https://www.terraform.io/enterprise/admin/infrastructure/admin) rotates the encryption password on active/active installations of Terraform Enterprise.
```
Comment (laurapacilio) [inline content/enterprise/releases/2022/v202201-2.mdx:35]: same here - should Active/Active be capitalized?

**Review by laurapacilio — CHANGES_REQUESTED** [review]
A couple of additional nits/questions. We're close!! Thank you again for the opportunity to review!

File: content/enterprise/admin/infrastructure/admin.mdx (hunk 5 above)
Comment (kosyfrances) [reply content/enterprise/admin/infrastructure/admin.mdx:7]: For standalone, the Replicated Admin Console is not disabled so I think leaving it just like your suggested change is fine.

**Review by laurapacilio — APPROVED** [review]
This looks good to me! Thank you for your work here!

File: content/enterprise/admin/infrastructure/admin-cli.mdx [hunk 10]
```diff
@@ -22,7 +18,9 @@ Note that `tfe-admin` is an alias for `replicated admin`, and can be used interc
 tfe-admin support-bundle
 ```
 
-This command generates a support bundle for all nodes. The support bundles will be uploaded to the same object store bucket that is used to store Terraform state files. The support bundles for a specific run of the admin command will all be uploaded to a directory with the same JobID (a timestamp in [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) format). If you are sending a support bundle to HashiCorp Support, package and send all associated bundles to ensure that we have all the necessary information.
+This command generates a support bundle for all nodes in `/var/lib/replicated/support-bundles`.
```
Comment (jmariedee) [inline content/enterprise/admin/infrastructure/admin-cli.mdx:21]: This reads (to me) as though support bundles are generated for nodes that exist in that file path. Could we rephrase tobreak this up?

"This command generates a support bundle for all nodes.

For Standalone (single node) installations the support bundle will be created in `/var/lib/replicated/support-bundles`.

File: content/enterprise/releases/2022/v202201-2.mdx [hunk 11]
```diff
@@ -32,7 +32,7 @@ APPLICATION LEVEL FEATURES:
 1. Container resource usage metrics are now available in Prometheus format.
 1. The private registry UI now displays a warning message for old versions of provider documentation with a link to the latest version. It also includes an **On this Page** outline for provider documentation that lets you navigate more quickly between sections. 
 1. Added an outline to the public provider documentation page
-1. The [new `tfe-admin rotate-encryption-password` command](https://www.terraform.io/enterprise/admin/infrastructure/active-active) rotates the encryption password on active/active installations of Terraform Enterprise.
```
Comment (jmariedee) [inline content/enterprise/releases/2022/v202201-2.mdx:35]: Same question. If the link redirects let's leave the old release notes that introduced this feature untouched.

**Review by jmariedee — APPROVED** [review]
LGTM!
